### PR TITLE
[EASI-4501] MDM to MDM-POR Changes

### PR DESCRIPTION
**Pull request by OddTomBrooks**
# EASI-4501

## Description

- Renamed `MDM` to `MDM-POR` in migration and gql schema
- Updated translations to match
- Added support for `Trans` component rendered links in timeline
- Updated unit tests

## How to test this change

- Verify name changes match Figma

## PR Author Checklist

- [ ] I have provided a detailed description of the changes in this PR.
- [ ] I have provided clear instructions on how to test the changes in this PR.
- [ ] I have updated tests or written new tests as appropriate in this PR.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.

### CODE REVIEW

File: migrations/V162__Update_MDM_To_MDM_POR.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ALTER TYPE OPERATIONAL_SOLUTION_KEY RENAME VALUE 'MDM' TO 'MDM-POR';
+
+UPDATE possible_operational_solution
+SET sol_name = 'Master Data Management Program-Organization Relationship',
+    sol_key = 'MDM-POR',
```
Comment (StevenWadeOddball) [inline migrations/V162__Update_MDM_To_MDM_POR.sql:5]: This line (sol_key = 'MDM-POR',) isn't needed, the alter type expression already renames the key.

File: pkg/graph/schema/types/shared_enums.graphql [hunk 2]
```diff
@@ -121,7 +121,7 @@ enum OperationalSolutionKey{
   INTERNAL_STAFF
   LDG
   LV
-  MDM
+  MDMPOR
```
Comment (StevenWadeOddball) [inline pkg/graph/schema/types/shared_enums.graphql:124]:
```suggest
MDM_POR
```

File: pkg/models/possible_operational_solution.go [hunk 3]
```diff
@@ -40,7 +40,7 @@ const (
 	OpSKInternalStaff             OperationalSolutionKey = "INTERNAL_STAFF"
 	OpSKLdg                       OperationalSolutionKey = "LDG"
 	OpSKLv                        OperationalSolutionKey = "LV"
-	OpSKMdm                       OperationalSolutionKey = "MDM"
+	OpSKMdmPor                    OperationalSolutionKey = "MDMPOR"
```
Comment (StevenWadeOddball) [inline pkg/models/possible_operational_solution.go:43]:
```suggest
OpSKMdmPor                    OperationalSolutionKey = "MDM_POR"
````

File: src/i18n/en-US/modelPlan/solutions.ts [hunk 4]
```diff
@@ -107,7 +107,7 @@ const operationalSolutions: TranslationOperationalSolutions = {
       INTERNAL_STAFF: 'Internal staff',
       LDG: 'Learning and Diffusion Group',
       LV: 'Legal Vertical',
-      MDM: 'Master Data Management',
+      MDMPOR: 'Master Data Management Program-Organization Relationship',
```
Comment (StevenWadeOddball) [inline src/i18n/en-US/modelPlan/solutions.ts:110]: See other notes, I suggest we rename the key to MDM_POR

File: migrations/V162__Update_MDM_To_MDM_POR.sql [hunk 5]
```diff
@@ -0,0 +1,8 @@
+ALTER TYPE OPERATIONAL_SOLUTION_KEY RENAME VALUE 'MDM' TO 'MDM-POR';
```
Comment (StevenWadeOddball) [inline migrations/V162__Update_MDM_To_MDM_POR.sql:1]:
```suggestion
ALTER TYPE OPERATIONAL_SOLUTION_KEY RENAME VALUE 'MDM' TO 'MDM_POR';
```
I think that `_` might be more consistent

**Review by StevenWadeOddball — CHANGES_REQUESTED** [review]
I have not reviewed the front end code, but I wanted to provide some feedback regarding backend migrations.

**Review by StevenWadeOddball — APPROVED** [review]
Thanks @patrickseguraoddball and @OddTomBrooks. I made a few simple fixes to the backend code, and verified the frontend matches figma. This should be good to go! Can you please take a quick look at the backend changes I made before merging?

**Review by patrickseguraoddball — APPROVED** [review]

**Pull request merged**